File: src/test/java/TC/김서방Test.java
package TC;

import org.assertj.core.api.Assertions;
import org.example.TC.김서방;
import org.junit.jupiter.api.Test;

public class 김서방Test {
    private final 김서방 kim = new 김서방();

    @Test
    void solution() {
        String[] seoul = {"Jane", "Kim"};
        String answer = "김서방은 1에 있다";
        String result = kim.solution(seoul);
        Assertions.assertThat(answer).isEqualTo(result);
    }
}

File: src/test/java/TC(K)/김서방Test.kt
package `TC(K)`

import io.kotest.core.spec.style.StringSpec
import org.assertj.core.api.Assertions
import org.example.TCK.김서방

class 김서방Test: StringSpec(
    {
        "김서방 찾기"{
            val seoul = arrayOf("Jane", "Kim")
            val answer = "김서방은 1에 있다"
            val result = 김서방().solution(seoul)
            Assertions.assertThat(answer).isEqualTo(result)
        }
    }
)